types/gsi/tslint.json

```diff
@@ -0,0 +1,7 @@
+{
+    "extends": "@definitelytyped/dtslint/dt.json",
+    "rules": {
+        "dt-header": false,
+        "npm-naming": false
```

Comment:
Remove these rules and fix errors.

Reply:
thank you for adding a review. what do you suggest we add as a version number? i've been unable to find a version number in the api's official documentation.